### DISCUSSION TITLE
Inconsistency in README.md

### DISCUSSION THREAD

**Discussion by AlfredSK** [minio/minio#12686] [Q&A] [answered]
While trying to figure out how to access the 'console' if MinIO is running behind a proxy I discovered an inconsistency in the README.md chapter about the console.

> ### Things to consider
> MinIO redirects browser access requests to the configured server port (i.e. 127.0.0.1:9000) to the configured Console port. MinIO uses the hostname or IP address specified in the request when building the redirect URI. The URL and port must be accessible by the client for the redirection to work
> 
> For deployments behind a load balancer, proxy, or ingress rule where the MinIO host IP address or port is not public, use the **MINIO_BROWSER_REDIRECT_URI** environment variable to specify the external hostname for the redirect. The LB/Proxy must have rules for directing traffic to the Console port specifically.
> 
> For example, consider a MinIO deployment behind a proxy https://minio.example.net, https://console.minio.example.net with rules for forwarding traffic on port :9000 and :9001 to MinIO and the MinIO Console respectively on the internal network. Set **MINIO_BROWSER_REDIRECT_URL** to https://console.minio.example.net to ensure the browser receives a valid reachable URL.

See? At first the config variable is called `..._URI`. But later it is called `..._URL`.

Which one is it?

**Comment by harshavardhana** [comment, marked as answer]
Its MINIO_BROWSER_REDIRECT_URL
